gammapy/irf/background.py

```diff
@@ -134,6 +134,32 @@ def to_fits(self, name='BACKGROUND'):
         """Convert to `~astropy.io.fits.BinTable`."""
         return fits.BinTableHDU(self.to_table(), name=name)
 
+    def evaluate(self, detx, dety, energy_reco, method=None, **kwargs):
+        """
+        Evaluate the `Background3D` at a given FOV coordinate and energy. The coordinates det_x, det_y and erngy_reco
+        should have the same dimension that match the numbers of point you want to evaluate
```

Comment:
I think you also need an empty line here above the `Parameters` section for the docs build to work.